tests/base.py

```diff
@@ -25,7 +25,10 @@ class SnapchatBase(unittest.TestCase):
     INCREMENTAL = "INCREMENTAL"
     OBEYS_START_DATE = "obey-start-date"
 
+    # TDL-21946 Work-Item to generate test data for following list of streams
     stats_streams = {"ad_account_stats_daily", "ad_account_stats_hourly", "pixel_domain_stats", "campaign_stats_daily", "campaign_stats_hourly", "ad_squad_stats_daily", "ad_squad_stats_hourly", "ad_stats_daily",  "ad_stats_hourly"}
```

Comment:
Line is too long, refactor it.

Reply:
Made changes but this line is not modified in this PR and refactoring is not scope this PR